src/Resources/skeleton/security/Voter.tpl.php

```diff
@@ -10,10 +10,13 @@ class <?= $class_name ?> extends Voter
 {
     protected function supports($attribute, $subject)
     {
-        // replace with your own logic
-        // https://symfony.com/doc/current/security/voters.html
-        return in_array($attribute, ['POST_EDIT', 'POST_VIEW'])
-            && $subject instanceof \App\Entity\BlogPost;
+        // Replace with your own logic
+        // See https://symfony.com/doc/current/security/voters.html
+        //
+        // return in_array($attribute, ['POST_EDIT', 'POST_VIEW'])
```

Comment:
Maybe replace POST by something else? It means "after" also, this can be confusing.

Reply:
Good catch. At the very least, we could change this to `BLOG_POST_EDIT`... though there is also no rule that says that things need to be upper case...

If we add a question about the entity/class, then we would derive this from the entity class name.